script/index.js

```diff
@@ -0,0 +1,272 @@
+const header = document.querySelector('.header');
+const headerBody = document.querySelector('.header-body');
+const hambergerIcon = document.querySelector('#hamberger');
+const iconMenu = document.querySelector('#icon-menu');
+const iconCross = document.querySelector('#icon-cross');
+const artCardContainer = document.querySelector('.main-container__art');
+const showMore = document.querySelector('.main-container__detail');
+const speakersContainer = document.querySelector('.featured-speakers');
+
+// //////////////////// Sticky NavBar
+const navbar = document.querySelector('.header-nav');
+const sticky = navbar.offsetTop;
+
+window.onscroll = () => {
+  if (window.pageYOffset >= sticky) {
+    navbar.classList.add('sticky');
+    headerBody.classList.add('content');
+  } else {
+    navbar.classList.remove('sticky');
+    headerBody.classList.remove('content');
+  }
+};
+
+// /////////////////// Mobile Menu
+const ul = document.createElement('ul');
+ul.classList.add('header-nav-container__nav-bar--link', 'nav-mobile');
+const navItems = [
+  {
+    name: 'About',
+    href: '../pages/about.html',
+  },
```

Comment:
- [ ] Please double-check on the mobile menu about page link because it seems to be broken as it reaches a 404 not found page! 🙏